### ISSUE TITLE
How to get Softmax layer?

### ISSUE THREAD

**Issue by Firyuza** [allanzelener/YAD2K#56]
In **def darknet19** I see **logits = DarknetConv2D(1000, (1, 1), activation='softmax')(body)**
Should I insert code to this place to get Softmax layer or it already exists?

Thanks.

**Comment by Firyuza** [comment]
Done!
